MIGRATION.md

```diff
@@ -0,0 +1,155 @@
+# v0.64.0
+
+This guide documents the removal of deprecated nodes from the Griptape Nodes Advanced Media Library in version 0.64.0.
+
+## Overview
+
+Version 0.64.0 removes deprecated nodes that were previously marked for removal. These nodes have been replaced with more flexible and powerful alternatives, primarily the new Diffusion Pipeline Builder system.
+
+### Affected Libraries
+
+| Library                               | Version |
+| ------------------------------------- | ------- |
+| Griptape Nodes Advanced Media Library | 0.64.0  |
+
+## Removed Nodes and Replacements
+
+### Image Processing Nodes
+
+| Display Name  | Replacement                                                                                                                                          |
+| ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Desaturate    | `Grayscale Image` from Griptape Nodes Library<br/>Location: `Image/Edit/Grayscale Image`<br/>[See details ↓](#for-image-processing-nodes)            |
```

Comment:
Can we add some example images of these nodes?

<img width="1134" height="906" alt="Image" src="https://github.com/user-attachments/assets/e3799cb9-bb79-485a-8446-aca639b66aa7" />

<img width="1124" height="926" alt="Image" src="https://github.com/user-attachments/assets/8a602390-bfd5-4452-873f-0f8e94cc292e" />

<img width="1258" height="931" alt="Image" src="https://github.com/user-attachments/assets/925cb3e5-cda5-44b0-9132-99e74e98dda2" />